app/templates/tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true
   },
+  "typeRoots": [
```

Comment:
isn't this default by typescript? As I'm aware of, @types is the default folder to look for typings, do you really need to provide the `typeRoots` info?